### PR TITLE
Docker compose frontend

### PR DESCRIPTION
**Pull request by bmath10**
**Note**: This PR is NOT dependent on https://github.com/onc-healthit/lantern-front-end/pull/11.
- Test by running `docker-compose up` and opening `localhost:8090`.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: https://oncprojectracking.healthit.gov/support/browse/LANTERN-93
  - vendor/ directories have been updated
    - New dependencies have been added
    - Local dependencies have been updated
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:** @mayerm94 

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
  - vendor/ directories have been updated
    - New dependencies have been added
    - Local dependencies have been updated
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge is complete.

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -18,4 +18,7 @@ services:
   lantern-mq:
     ports:
       - "15672:15672"
-      - "5672:5672"
\ No newline at end of file
+      - "5672:5672"
+  lantern_front_end:
+    ports:
```
Comment (mayerm94) [inline docker-compose.override.yml]: Can remove the lantern_front_end entry from the override since it is duplicating what is in the docker-compose.

Comment (bmath10) [reply docker-compose.override.yml]: 👍

File: docker-compose.yml [hunk 2]
```diff
@@ -56,6 +56,12 @@ services:
     container_name: lantern-mq
     image: rabbitmq:3-management
 
+  lantern_front_end:
+    container_name: lantern_front_end
+    image: onchealthit/lantern-front-end
```
Comment (edeyoung) [inline docker-compose.yml]:
```suggestion
    image: onchealthit/lantern-front-end
    environment:
      - LANTERN_<frontend url>=<frontend url>
```

Comment (bmath10) [reply docker-compose.yml]: I think the issue here was actually that the image on the docker-hub repo needed to be updated, so im waiting for confirmation that that was the fix!

**Pull request merged**